<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>backgroundSubtraction</class>
 <widget class="QDialog" name="backgroundSubtraction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>293</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>backgroundSubtraction</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>height</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="hRect"/>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Radius of inner circle</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QPushButton" name="applyButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Apply background subtraction</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="innerRadiusLineEdit">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="outerRadiusLineEdit">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="recPushButton">
       <property name="text">
        <string>Create rectangle subtraction Aux</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Radius of outer circle</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>XCenter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="rectXCenter"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>YCenter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="rectYCenter"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>width</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="wRect"/>
       </item>
      </layout>
     </item>
     <item row="0" column="1" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>X Center</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="xCenterLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Y Center</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="yCenterLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
